config_role.go

```diff
@@ -0,0 +1,96 @@
+package evergreen
+
+import (
+	"time"
+
+	"github.com/mongodb/anser/bsonutil"
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// LDAPRoleMap contains a mapping of LDAP groups for a user to roles. LDAP
+// groups are represented by their name and roles by their unique ID in the
+// roles collection.
+type LDAPRoleMap struct {
+	LastUpdated time.Time         `bson:"last_updated" json:"last_updated" yaml:"last_updated"`
```

Comment:
Do you have a use in mind for this? If the purpose is an audit trail, we have that for the admin struct already